Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -286,80 +291,85 @@ public float AnimationLength
             set { animationLength = value; }
         }
 
-        [Tooltip("Number of items in a row on up-down scroll direction or number of items in a column on left-right scroll direction.")]
+        [Tooltip("Number of cells in a row on up-down scroll view or number of cells in a column on left-right scroll view.")]
         [SerializeField]
         [FormerlySerializedAs("tiers")]
-        private int itemsPerTier = 1;
+        private int cellsPerTier = 1;
 
         /// <summary>
-        /// Number of items in a row on up-down scroll direction or number of items in a column on left-right scroll direction.
+        /// Number of cells in a row on up-down scroll or number of cells in a column on left-right scroll.
         /// </summary>
-        public int ItemsPerTier
+        public int CellsPerTier
         {
             get
             {
-                Debug.Assert(itemsPerTier > 0);
-                return itemsPerTier;
+                Debug.Assert(cellsPerTier > 0);
```

Comment:
>Debug.Assert(cellsPerTier > 0); [](start = 16, length = 31)

i would move the assert into set - also is this later on going to be used for divisions? if so should we just clamp this to 1?